### PR TITLE
Added playback_state config option.

### PR DESCRIPTION
**Pull request by Bettehem**
Hello!
I added a playback_state config option. It can be used to pre-define a certain playback state for the player, for example:
`playback_state = "playing"` - Start playing the current song when ncspot is opened
`playback_state = "stopped"` - Stop the current song when ncspot is opened

### CODE REVIEW

**Review by hrkfdn — COMMENTED** [review]
Thanks for the PR and sorry for the delay. I have some minor comments, WDYT?

File: src/config.rs [hunk 1]
```diff
@@ -129,6 +132,16 @@ impl Config {
...
         }
 
+        if let Some(playback_state) = values.playback_state.clone() {
+            let state = playback_state.to_lowercase();
+            match state.to_lowercase().as_str() {
+                "stopped" => userstate.playback_state = state.to_string(),
+                "playing" => userstate.playback_state = state.to_string(),
+                "paused" => userstate.playback_state = state.to_string(),
+                "default" | _ => userstate.playback_state = String::from("default"),
+            }
```
[the remark this answers is not in the document]
Comment (Bettehem) [reply src/config.rs:142]: Yeah, perhaps using `enum`s would be better🤔

File: src/queue.rs [hunk 2]
```diff
@@ -42,21 +42,35 @@ pub struct Queue {
...
             spotify.update_track();
-            spotify.pause();
-            spotify.seek(state.track_progress.as_millis() as u32);
+            match playback_state.as_str() {
+                "stopped" => {
+                    spotify.stop();
+                }
+                "playing" => {
+                    spotify.play();
+                }
```
[the remark this answers is not in the document]
Comment (Bettehem) [reply src/queue.rs:69]: Ah, good catch! I'll fix this too.

**Comment by Bettehem** [comment]
Hi! Thanks for the comments, I have now made the changes you suggested. How does it look?

**Comment by hrkfdn** [comment]
Cool, thanks, let's get it in! :)

**Pull request merged**